cdserver/app/models/building.rb

```diff
@@ -0,0 +1,8 @@
+# a building that houses shelter units
```

Comment:
Please look for a gem for capturing physical addresses.  We need to model the physical address of a shelter building, but there is no need to reinvent the wheel.  BTW Physical address can be added as step two.  For now, a shelter building should have a name attribute.  The name is optional.  The name getter should automatically grab the street address if there is no specific building name.  you should add unit tests for this to make sure the defaulting behavior is working correctly.